tests/integration/targets/terraform/tasks/main.yml

```
@@ -0,0 +1,81 @@
+---
+
+# Fallback in case existing_terraform_path var is missing. Saves us from checking for `not defined`
+# conditions in multiple tasks below.
+
+- name: Check if existing_terraform_path var is missing
+  set_fact:
+      existing_terraform_path: "{{ existing_terraform_path | default('') }}"
```

How about simply setting this to an empty string?
```suggestion
      existing_terraform_path: ""
```
Tests shouldn't really accept input from users anyway. Either the correct version is found in $PATH, or otherwise it should download the correct version and use the downloaded one.

---

OK, I have reverted to prior behavior and tested once again. Both of those use cases are working. 

1. Terraform exists in $PATH and is at correct version.
2. Terraform does not exists in $PATH, and is downloaded from the vendor.